core/matrix/dense_kernels.hpp

```diff
@@ -71,45 +73,66 @@ namespace kernels {
                      const matrix::Dense<_type> *y,               \
                      matrix::Dense<_type> *result)
 
+
+#define GKO_DECLARE_DENSE_CONVERT_TO_COO_KERNEL(_type, _prec)        \
+    void convert_to_coo(std::shared_ptr<const DefaultExecutor> exec, \
+                        matrix::Coo<_type, _prec> *other,            \
+                        const matrix::Dense<_type> *source)
+
+
+#define GKO_DECLARE_DENSE_MOVE_TO_COO_KERNEL(_type, _prec)        \
+    void move_to_coo(std::shared_ptr<const DefaultExecutor> exec, \
+                     matrix::Coo<_type, _prec> *other,            \
+                     const matrix::Dense<_type> *source)
+
+
 #define GKO_DECLARE_DENSE_CONVERT_TO_CSR_KERNEL(_type, _prec)        \
     void convert_to_csr(std::shared_ptr<const DefaultExecutor> exec, \
                         matrix::Csr<_type, _prec> *other,            \
                         const matrix::Dense<_type> *source)
 
+
```

Comment:
nit: why the extra whitespace in this file?

All kernel declaration files use only 1 blank line between the two `#define`s, since they only list a bunch of functions.

Reply:
I got it.